tests/phpunit/tests/option/determineOptionAutoloadValue.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * @group option
+ */
+class Tests_Determine_Option_Autoload_Value extends WP_UnitTestCase {
+	public function set_up() {
+		add_filter( 'max_option_size', array( $this, 'filter_max_option_size' ) );
+		parent::set_up();
```

Comment:
This needs to be reversed, otherwise this hook will not be properly cleaned up after the tests.
```suggestion
		parent::set_up();
		add_filter( 'max_option_size', array( $this, 'filter_max_option_size' ) );
```